src/components/EmailEditor/store/emailEditorConfig.js

```diff
@@ -385,7 +428,39 @@ export default {
 	modules: {
 		BlockLibData,
 	},
-	state: getDefaultState(),
+	state: {
+		...getDefaultState(),
+		showExternalElementIconModal: false,
+		showExternalElementSocialMediaEmbedModal: false,
+		promiseResolver: null,
+		promiseRejecter: null,
+		parentWidth: 600,
+		externalElementJson: {},
+		externalElementIconData: {
+			iconType: "",
+			iconName: "",
+			iconSize: "",
+			iconStyle: "",
+			iconColorName: "",
+			displayIconColor: "",
+			selectedIcon: "",
+			isIconUpdate: false,
+		},
+		externalElementSocialImageData: {
+			sourceUrl: "",
+			platform: "",
+			postId: "",
+			theme: "light",
+			page: "",
+			imageWidth: 32,
+		},
+		openPeepsIconsArray: {
+			monochromatic: [],
+			multicolor: [],
+		},
+		humaansIconsArray: [],
+		externalElementsBackendBaseUrl: "https://external-elements.chamaileon.io",
```

Comment:
One idea:
put in a conditional to this like we have with the env select (where it selects if it should load the staging env or prod env) and use either external-elements or external-elements-staging so that we can fully test it on staging as well